### PR TITLE
readme changes

### PR DESCRIPTION
**Pull request by ndemou**
I've made two changes a) rephrased the introduction to be more clear and simple b) made a lot of minor grammar and syntactic fixes (and a two or maybe three small changes to the original sentences)

(I should have probably made 2 pull requests but hey I'm a github noob)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -157,20 +159,20 @@ Configuring anycast-healthchecker
...
+from the command line.
 
 Bird configuration
 ##################
 
 Below is an example configuration for Bird which establishes the logic described
-`How anycast-healthchecker works`_. It is the minimum configuration required by
-anycast-healthchecker to function properly.
+in `How anycast-healthchecker works`_. It is the minimum configuration required 
+to interface Bird and anycast-healthchecker.
```
Comment (unixsurfer) [inline README.rst]: 'to interface Bird and anycast-healthchecker' , I don't quite understand that

File: README.rst [hunk 2]
```diff
@@ -13,27 +13,29 @@ anycast-healthchecker
 Introduction
 ------------
 
-**anycast-healthchecker** is a Python program to monitor a service and instruct
-`Bird`_ daemon to advertise or withdraw a route associated with the service
-based on the result of a health check. It uses the Python `daemon`_ library to
-implement a well-behaved Unix daemon process and it also utilizes threading for
-running each service check.
+**anycast-healthchecker** monitors a service by doing 
+peridic health checks and based on the result instructing `Bird`_ daemon
```
Comment (unixsurfer) [inline README.rst]: s/peridic/periodic/,  right ?

File: README.rst [hunk 3]
```diff
@@ -13,27 +13,29 @@ anycast-healthchecker
...
 
-**anycast-healthchecker** is a Python program to monitor a service and instruct
-`Bird`_ daemon to advertise or withdraw a route associated with the service
-based on the result of a health check. It uses the Python `daemon`_ library to
-implement a well-behaved Unix daemon process and it also utilizes threading for
-running each service check.
+**anycast-healthchecker** monitors a service by doing 
+peridic health checks and based on the result instructing `Bird`_ daemon 
+to either advertise or withdraw the coresponding route. As a result Bird 
+will only advertise routes to healthy services.
```
Comment (unixsurfer) [inline README.rst]: s/to healthy services/for healthy services/

**Comment by unixsurfer** [comment]
Most of the changes will be merged.
Thanks a lot ndemou for your time, very much appreciated

File: README.rst (hunk 1 above)
Comment (ndemou) [inline README.rst]: my bad here - the correct form is "interface with" see https://www.google.gr/search?num=20&hl=el&biw=1076&bih=970&q=%22interface+with%22+english+dictionary&oq=%22interface+with%22+english+dictionary&gs_l=serp.3...41601.52739.0.52960.26.25.0.0.0.0.290.2504.0j17j1.18.0....0...1c.1.64.serp..13.13.1878.zqIsGYybtFM

File: README.rst (hunk 2 above)
Comment (ndemou) [inline README.rst]: yes!

File: README.rst (hunk 3 above)
Comment (ndemou) [inline README.rst]: whatever -- both seem equally good to me

**Pull request merged**